### PR TITLE
Update geckoboard.md

### PR DESCRIPTION
**Pull request by Nirvikalpa108**
This is a modified version of the [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md).

This pull request adheres to the repository's [Code of Conduct](https://github.com/remoteintech/remote-jobs/blob/master/CODE_OF_CONDUCT.md).

- [ ] I am an employee of the company mentioned and confirm all included details are correct
- [ ] This PR contains housekeeping only (URL edits, copy changes etc)
- [ ] You know your alphabet - company is listed in alphabetical order in the README
- [ ] The company directly hires employees. No bootcamps / freelance sites / etc
- [ ] The company added hires remote employees, or positions are available to remote workers and are clearly illustrated as such
- [ ] A [company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md) is included - __Required__ for new additions. (This can be a basic outline but at least something please)
- [ ] __Remote status__ has details regarding how the culture includes remote employees, how the company integrated remote workers, etc
- [ ] __Region__ details any restrictions to applicants based on geography
- [ ] __How to apply__ details the best approach for new applications, page on site where open position are listed, and any other help available for job hunters

### CODE REVIEW

**Comment by auto-nylen** [comment]
Hi @Nirvikalpa108, thanks for your contribution!

Our automated checks detected some problems with this pull request (as of commit 78ef605).  See below for details:

```
geckoboard.md: Profile looks complete, but the main readme contains a warning emoji.

1 problem detected
```

Please fix these issues and push a new commit to this pull request.

File: company-profiles/geckoboard.md [hunk 1]
```diff
@@ -2,10 +2,28 @@
...
-- Follow the structure of the example profile and [send us a pull request with your changes to this file!](https://github.com/remoteintech/remote-jobs/edit/master/company-profiles/geckoboard.md)
+20-50
+
+## Remote status
+
+We believe work should fit around your life, which means no overtime and the flexibility to work from home when you need to.
+
+## Region
+
+Headquartered in London, but with team members across the world.
```
Comment (dougaitken) [inline company-profiles/geckoboard.md]: If the company is open to hiring globally, please make this `Worldwide`. The location is under Office Locations as you've mentioned

**Comment by dougaitken** [comment]
Hey @Nirvikalpa108 - thanks for opening this Pull Request to update the Geckoboard listing!

What the bot is complaining about is that in the main [README.md](https://github.com/remoteintech/remote-jobs/blob/master/README.md) file, you'll see the ⚠️emoji. We use this to indicate that a company doesn't have a profile. So now you've added the profile, the bot is confused why the emoji is still there.

geckoboard_warning

If you can edit your pull request to remove that ⚠️from next to geckoboard on the main README file, and then link to the company profile, we'll get it merged.

If you're not sure how, just let us know

**Comment by auto-nylen** [comment]
Hi @nylen, thanks for updating the pull request!  According to our automated checks, everything looks OK now (as of commit cf43752). :heavy_check_mark:

**Comment by nylen** [comment]
Thanks for the PR!  I've fixed up the readme and the region field.

**Comment by Nirvikalpa108** [comment]
Hi @dougaitken, I just want to say a huge thank you for responding to my question and teaching me how to address the error message. I'll try to practice changing the README file along with the company profile for next time. Thank you very much indeed.

Thanks for fixing @nylen. I'm not sure about the wording in the region field which is currently "Worldwide, with team members across the world." It seems to be a repetition. Perhaps just having the world "Worldwide" alone would be sufficient? It's a small point, so understand if you prefer not to change! 
Thank you again!

**Comment by nylen** [comment]
Huh, you're right, I didn't do a great job there. I'll fix it tomorrow, or if you want to try your hand at another PR, I agree it should just be Worldwide.

The Region field in general is a bit messy at the moment. To me there is not a good reason to have this in both the readme and the company profile. We're discussing this more general issue at #485 and would love to hear what you and others think too.

**Pull request merged**